core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/RegionExecutionCoordinator.scala

```diff
@@ -194,7 +198,9 @@ class RegionExecutionCoordinator(
 
   private def sendStarts(region: Region): Future[Seq[Unit]] = {
     asyncRPCClient.sendToClient(
-      ExecutionStatsUpdate(workflowExecution.getRegionExecution(region.id).getStats)
+      ExecutionStatsUpdate(
+        workflowExecution.getAllRegionExecutionsStats
```

Comment:
same here. are we sending too much information (more than needed) to the frontend?

Reply:
https://github.com/Texera/texera/pull/2570#discussion_r1554849374